### PR TITLE
Mvp

### PR DESCRIPTION
**Pull request by DrewGregory**
This is the basic beginning of the MVP for DollarForEveryone.

It includes:

- 2 screens overlayed over the Burner Wallet: the Splash screen with some explanatory text and a deep link button, NotVerified Screen which encourages the user that they have linked their BrightID but need to do some extra work to reach verified status. Once the user is linked and verified, they are automatically navigated to the burner wallet screen.
- Backend server code: This receives the autogenerated client address for the burner wallet, which will then return a deep link associated with the address. The same endpoint will also check if the context id is already linked and respond with the appropriate status. If the server learns for the first time that the context id is now with a verified brightid, it will send $1 to the associated eth address.

### CODE REVIEW

**Comment by DrewGregory** [comment]
This does *not* cover various aspects of the MVP including the feedback form and privacy policy.

File: backend/server.js [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+  }
+  
+}
+
+const sendDollar = async (address) => {
+  const txConfig = {
+    to: address,
+    value: web3.util.toHex(web3.util.toWei(1)), 
+    gas: 21000,
+    gasPrice: 100000000,
```
Comment (adamstallard) [inline backend/server.js:164]:
```suggestion
    gasPrice: 10000000000,
```

File: backend/server.js [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+  };
+  return s.replace(/[/+=]/g, (c) => alts[c]);
+}
+
+function hash (data)  {
+  const h = CryptoJS.SHA256(data);
+  const b = h.toString(CryptoJS.enc.Base64);
+  return b64ToUrlSafeB64(b);
+};
+
```
Comment (adamstallard) [inline backend/server.js:68]:
```suggestion
```

It seems this function is no longer used now that we're using stable-json-stringify and nacl.sign to encode operations.

File: backend/server.js [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+}
+
+const sendDollar = async (address) => {
+  const txConfig = {
+    to: address,
+    value: web3.util.toHex(web3.util.toWei(1)), 
+    gas: 21000,
+    gasPrice: 100000000,
+    nonce: await web3.eth.getTransactionCount(web3.eth.defaultAccount),
+    chainID: 1337 // ganache chain id
```
Comment (adamstallard) [inline backend/server.js:166]: We'll probably need a way to set the real chainID for IDChain (`0x4a` or `74`) and connect to an rpc node (https://idchain.one/rpc)  See "[add IDChain to metamask](https://brightid.gitbook.io/brightid/idchain/add-idchain-to-metamask)"

File: basic-wallet/src/App.tsx [hunk 4]
```diff
@@ -0,0 +1,95 @@
+
+import React, { useState, useEffect } from 'react'
+
+import * as serviceWorker from './serviceWorker';
+import { xdai, eth } from '@burner-wallet/assets';
+import BurnerCore from '@burner-wallet/core';
+import { InjectedSigner, LocalSigner } from '@burner-wallet/core/signers';
+import { InfuraGateway, InjectedGateway, XDaiGateway, } from '@burner-wallet/core/gateways';
+import Exchange, { Uniswap, XDaiBridge } from '@burner-wallet/exchange';
```
Comment (adamstallard) [inline basic-wallet/src/App.tsx:9]: We may need to look at the xdai gateway, xdai bridge, and uniswap libraries and make sure we're using the idchain equivalents, e.g. https://uniswap.idchain.one https://dai.idchain.one

**Review by adamstallard — CHANGES_REQUESTED** [review]
The only substantial changes involve making sure that we're connecting to IDChain, not xdai chain.  Happy to help with this.

**Pull request closed without merging**